packages/app/src/files-manager/components/upload-files-modals/upload-files-modals.component.tsx

```diff
@@ -82,15 +83,17 @@ export const UploadFilesModals: FunctionComponent<UploadFilesModalProps> = ({
           testId={UploadFilesModalsTestIds.UploadedPopUp}
         />
       )}
-      {uploadingFailed && (
-        <ErrorModal
-          testId={UploadFilesModalsTestIds.ErrorModal}
-          open={uploadingFailed}
-          title={intl.formatMessage(errorTitle)}
-          subtitle={intl.formatMessage(errorSubtitle)}
-          closeModal={onCloseUploadingErrorModal}
-        />
-      )}
+      {uploadingFailed &&
+        error?.type !== FilesManagerError.UploadDuplicates && (
```

Comment:
what if the type of error is another one? The application displays some error or can't be happened right now?

Reply:
All other error types are shown with a general error modal, as it has been so far